src/config.js

```diff
@@ -174,8 +177,12 @@ module.exports.setConfig = function setConfig(configData) {
         config.disableHttpResponseBodyCapture = configData.disableHttpResponseBodyCapture;
     }
 
-    if (configData.ignoredKeys) {
-        config.ignoredKeys = configData.ignoredKeys;
+    if (configData.ignoredKeys && Array.isArray(configData.ignoredKeys)) {
+        const filteredIgnoredKeys = configData.ignoredKeys.filter(key => typeof key !== 'function');
```

Comment:
consider filtering explicitly only for strings & regexes, and reject everything else with a warning (not just functions).

Reply:
so the problem there is no clear way to filter a regexp (`typeof /wr32/ === 'object'`) - there are some hacks online but they are sketchy at best